pyop2/host.py

```diff
@@ -538,16 +538,22 @@ def c_map_bcs(self, sign):
             if not map.iterset._extruded:
                 continue
             for j, m in enumerate(map):
-                if 'bottom' not in m.implicit_bcs:
-                    continue
-                need_bottom = True
-                for idx in range(m.arity):
-                    if m.bottom_mask[idx] < 0:
-                        val.append("xtr_%(name)s[%(ind)s] %(sign)s= %(val)s;" %
-                                   {'name': self.c_map_name(i, j),
-                                    'val': max_int,
-                                    'ind': idx,
-                                    'sign': sign})
+                bottom_masks = None
```

Comment:
I disagree. Bottom_masks is never a list. When it's not None it's a numpy array.